universe/space/space_attributes.go

```diff
@@ -189,6 +189,10 @@ func (s *Space) UpsertSpaceAttribute(
 		}
 	}()
 
+	if s.world != nil {
+		s.world.GetCalendar().OnAttributeUpsert(attributeID, payload.Value)
+	}
```

Comment:
attributes logic was changed, see
https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/space/space_attributes.go#L395

Reply:
hooked registered in new place